tests/test.livecodescript

```diff
@@ -0,0 +1,90 @@
+script "TestMergJSON"
```

Comment:
Probably want this to be `TestMergMarkdown` :-)

Reply:
aha! yes